canonical-websites/www.ubuntu.com
# Remove server/plans-and-pricing page

## Done

* Removed server/plans-and-pricing page
* Fixed link on hyperscale and updated the [copy doc](https://docs.google.com/document/d/1f56CCzH519lIVa4KvxDt0QpxYFMK-bMdVlKo4TvPSIk/edit)
* Removed it from the navigation
* Added a redirect to /support/plans-and-pricing

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- see that Plans and pricing has been removed from the navigation
- see that [hyperscale]() links to  /support/plans-and-pricing
- see that [server/plans-and-pricing](http://0.0.0.0:8001/server/plans-and-pricing) goes to /support/plans-and-pricing

## Fixes

this was reviewed and approved in this [PR](https://github.com/canonical-websites/www.ubuntu.com/pull/2368)
